pygsheets/worksheet.py

```diff
@@ -1267,6 +1267,45 @@ def copy_to(self, spreadsheet_id):
         new_spreadsheet = self.client.open_by_key(spreadsheet_id)
         return new_spreadsheet[response['index']]
 
+
+    def sort_range(self,start,end,basecolumnindex,sortorder):
+        """Sorts the data in rows based on the given column index.
+
+        :param start:               Address of the starting cell of the grid (e.g.-> "A1") 
+        :param end:                 Address of the last cell of the grid to be considered (e.g.-> "B5")
+        :param basecolumnindex:     Index of the base column in which sorting is to be done (Integer)
```

Comment:
Index from where ? relative to the start address given or is it the index of column in spreadsheet?